src/domain/users/entities/invitation.entity.ts

```diff
@@ -6,5 +6,6 @@ export type Invitation = {
   userId: User['id'];
   orgId: Organization['id'];
   role: UserOrganization['role'];
+  name: UserOrganization['name'];
```

Comment:
Nit: this `name` field should be also included to the `route` entity (`/routes/organizations/entities/invitation.entity.ts`), otherwise it would be missing in the OpenAPI specification.